docs/ref/models/querysets.txt

```diff
@@ -2476,6 +2476,55 @@ Class method that returns an instance of :class:`~django.db.models.Manager`
 with a copy of the ``QuerySet``’s methods. See
 :ref:`create-manager-with-queryset-methods` for more details.
 
+
+``explain()``
+~~~~~~~~~~~~~
+
+.. versionadded: 2.1
+
+.. method:: explain(format=None, **options)
+
+Returns the execution plan of the current queryset as a string. This details
+how the database would execute the given query, highlighting any indexes or
+joins used. For example, when using PostgreSQL::
+
+    >>> print(Blog.objects.filter(title="My Blog").explain())
+    Seq Scan on blog  (cost=0.00..35.50 rows=10 width=12)
+      Filter: (title = 'My Blog'::bpchar)
+
+This method is currently supported by all built in database backends with the
```

Comment:
chop "currently". Is there any plan to add Oracle support? I didn't follow your discussion closely but I thought you basically determined it's infeasible.